### ISSUE TITLE
this is about youtube api for logging in and cancelling/subscribing premium subscription?

### ISSUE THREAD

**Issue by SouvikBarua** [spotify/web-api#1348]
So I basically have not idea about programming but I do know that through API this problem can be solved.

So I am looking to create a saas platform which will help users to subscribe/unsubscribe their youtube premium subscriptions whenever they want as well as pay their subscription amount via us.

Is there any feature of the Youtube API which can help me to solve this problem of mine? If yes then how to go forward with it in details?

**Comment by jscholes** [comment]
This is a Spotify API, nothing to do with YouTube.

**Comment by SouvikBarua** [comment]
my bad.

So I basically have not idea about programming but I do know that through API this problem can be solved.

So I am looking to create a saas platform which will help users to subscribe/unsubscribe their youtube premium subscriptions whenever they want as well as pay their subscription amount via us.

Is there any feature of the Spotify API which can help me to solve this problem of mine? If yes then how to go forward with it in details?

**Comment by SouvikBarua** [comment]
> This is a Spotify API, nothing to do with YouTube.

my bad.

So I basically have not idea about programming but I do know that through API this problem can be solved.

So I am looking to create a saas platform which will help users to subscribe/unsubscribe their spotify premium subscriptions whenever they want as well as pay their subscription amount via us.

Is there any feature of the Spotify API which can help me to solve this problem of mine? If yes then how to go forward with it in details?

**Comment by jscholes** [comment]
No, the Spotify Web API cannot be used to manage a user's subscription, nor to facilitate subscription payments.  Sorry.

**Comment by SouvikBarua** [comment]
Hi,

Can you tell me then how does ideel.com and https://www.truebill.com/ is able to do that?
